packages/react/src/components/pagination/Pagination.stories.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from 'react';
+
+import { Pagination } from './Pagination';
+
+export default {
+  component: Pagination,
+  title: 'Components/Pagination',
+  parameters: {
+    controls: { expanded: true },
+  },
+  args: {},
+};
+
+// args is required for docs tab to show source code
+// eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
+export const Basic = (args) => (
```

Comment:
It would be nice if the control tab's args would work, at least on some level

Reply:
This is hard because then the props do not show up in the storybook's story tab. Let's try to create at least one Playground story where the controls work:

- [ ] Create Playground story where storybook's controls work